### DISCUSSION TITLE
Alternate calendar (e.g. persian) / localization

### DISCUSSION THREAD

**Discussion by mimhdev** [nocodb/nocodb#7233] [Q&A]
Hi
I really appreciate your great work

I'm just looking for how to contribute/develop an alternate calendar/locale support for date, duration & ...

Thanks

**Comment by Sirivasv** [comment]
I found this section for persian locale for the GUI that is in Vue.js
https://github.com/nocodb/nocodb/blob/develop/packages/nc-gui/lang/fa.json

As how to tune the specifics of date, duration, etc. might be worth checking i18n [docs](https://vue-i18n.intlify.dev/)
as it is configured for nocodb here:
https://github.com/nocodb/nocodb/blob/6948a043d1a15cac7b2bab9aa097baddf95cbe30/packages/nc-gui/plugins/a.i18n.ts
